src/MallardMessageHandlers.Refit/MallardMessageHandlers.Refit.csproj

```diff
@@ -29,7 +29,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Refit" Version="6.0.1" />
+		<PackageReference Include="Refit" Version="8.0.0" />
```

Comment:
Bumping a dependency to a new major version is a breaking change so your commit must reflect this.

- Your conventional commit should start with `fix!:` since this is a security fix and because it's a breaking change.
- You need to update BREAKING_CHANGES.md to document the breaking change.

Reply:
It would be the first breaking change in this repository. Here's an example of another package that has had breaking changes to give you an idea: https://github.com/nventive/Chinook.DynamicMvvm/blob/main/BREAKING_CHANGES.md